core/src/main/java/org/kiwix/kiwixmobile/core/extensions/ViewExtensions.kt

```diff
@@ -37,6 +37,20 @@ fun View.snack(
   }.show()
 }
 
+fun View.snack(
+  string: String,
```

Comment:
string is not a great parameter name, i should match the snackbar api which I would guess is `message`

Reply:
@macgills renamed the variable.